### PR TITLE
Tests for retry_utils

### PR DESCRIPTION
**Pull request by erthalion**
One can argue about how necessary they are,
but at least I remembered how to do golang.

### CODE REVIEW

File: pkg/util/retryutil/retry_util.go [hunk 1]
```diff
@@ -5,14 +5,39 @@ import (
...
 	}
+	tick := &Ticker{time.NewTicker(interval)}
+	return RetryWorker(interval, timeout, f, tick)
+}
+
+func RetryWorker(
+	interval time.Duration,
+	timeout time.Duration,
+	f func() (bool, error),
+	tick RetryTicker) error {
```
Comment (mgomezch) [inline pkg/util/retryutil/retry_util.go]: I don't think this is too important, but maybe the function parameter should be at the end to make invocations with long hanging lambdas more readable.

Comment (erthalion) [reply pkg/util/retryutil/retry_util.go]: Yeah, it actually bothered me too a bit.

**Comment by erthalion** [comment]
:+1:

**Comment by mgomezch** [comment]
:+1:

**Pull request merged**